### PR TITLE
ARSN-444: Add downsample policy actions

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: lib/policyEvaluator/utils/actionMaps.ts [hunk 1]
```diff
@@ -233,6 +233,9 @@ const actionMapScuba = {
     AdminStopIngest: 'scuba:AdminStopIngest',
     AdminReadRaftCseq: 'scuba:AdminReadRaftCseq',
     AdminTriggerRepair: 'scuba:AdminTriggerRepair',
+    AdminStartDownsample: 'scuba:AdminStartDownsample',
+    AdminStopDownsample: 'scuba:AdminStopDownsample',
```
Comment (dvasilas) [inline lib/policyEvaluator/utils/actionMaps.ts:237]: I think pause/resume is more representative than start/stop, but let's keep it consistent with {start|stop}ingest for now and we can decide at some point before release if we want to change both.

**Review by dvasilas — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]

**Review by anurag4DSB — APPROVED** [review]
LGTM

**Pull request merged**